jwst/extract_2d/tests/test_grisms.py

```diff
@@ -150,6 +144,8 @@ def create_tso_wcsimage(filtername="F277W", subarray=False):
 
     hdul['sci'].data = np.ones((2, subsize, 2048))
     im = CubeModel(hdul)
+    im.meta.wcsinfo.siaf_xref_sci = 887.0
+    im.meta.wcsinfo.siaf_yref_sci = 35.0
```

Comment:
Is there a reason to hard-code these without a check on whether SDP has populated them?  Might be good to first check, and than have this as a fall-back, for data that hasn't gone through the new SDP running of `set_telescope_pointing` yet.

Reply:
What SDP? This test creates the data.

Reply:
Ah, nevermind.  I see this is in the test.